### PR TITLE
Total liquidity store 

### LINKED ISSUE #155: Make TotalLiquidity for all pools queryable
We should add a global variable in state that lists out all coins that are LP'd, summed across all pools. (So a single variable thats the sum of liquidity in every pool) This will require adding to this variable every time someone LP's or withdraws liquidity, and every time someone executes a trade.

This is being done to provide something easy to query for the UI.

We should also make this queryable.

### PR DESCRIPTION
**Pull request by antstalepresh**
closes #155

### CODE REVIEW

File: x/gamm/client/cli/cli_test.go [hunk 1]
```diff
@@ -921,6 +921,43 @@ func (s *IntegrationTestSuite) TestGetCmdTotalShare() {
...
+
+	testCases := []struct {
+		name      string
+		args      []string
+		expectErr bool
+	}{
+		{
+			"query total liquidity", // osmosisd query gamm total-liquidity
+			[]string{
+				"1",
```
Comment (ValarDragon) [inline x/gamm/client/cli/cli_test.go:935]: What is this `1` for?

File: x/gamm/client/cli/cli_test.go [hunk 2]
```diff
@@ -921,6 +921,43 @@ func (s *IntegrationTestSuite) TestGetCmdTotalShare() {
...
 }
 
+func (s *IntegrationTestSuite) TestGetCmdTotalLiquidity() {
+	val := s.network.Validators[0]
+
+	testCases := []struct {
+		name      string
+		args      []string
+		expectErr bool
+	}{
```
Comment (ValarDragon) [inline x/gamm/client/cli/cli_test.go:931]: Can we create a pool and add a test that the total liquidity is correct, or is that too inconvenient to do for the CLI tests?

**Review by ValarDragon — APPROVED** [review]
LGTM! Nice work on this

File: x/gamm/client/cli/cli_test.go (hunk 1 above)
Comment (antstalepresh) [reply x/gamm/client/cli/cli_test.go:935]: Removed

File: x/gamm/client/cli/cli_test.go (hunk 2 above)
Comment (antstalepresh) [reply x/gamm/client/cli/cli_test.go:931]: I think it's complex to do in CLI as CLI has lots of other tests doing in parallel.

**Pull request merged**